pages/shop.tsx

```diff
@@ -13,40 +13,35 @@ const Shop: FC<ShopProps> = ({
 
 }: ShopProps) => {
 
-    function showCat() {
-        if (document.getElementById('bycategory')) {
+    function showCos() {
+        if (document.getElementById('makeup')) {
             document.getElementById('browseall').style.display = 'none';
-            document.getElementById('browsecategory').style.display = 'flex';
+            document.getElementById('browsemakeup').style.display = 'flex';
+            document.getElementById('restart').style.display = 'flex';
         }
     }
-    function showCos() {
-        if (document.getElementById('cosmetics')) {
-            document.getElementById('browsecategory').style.display = 'none';
-            document.getElementById('browsecosmetics').style.display = 'flex';
-        }
+    function restartBrowse() {
+        window.location.reload();
     }
     return (
         <div>
             <NavBar/>
             <MainContentContainer css="flex flex-col items-center h-5/6 mt-10">
                 <MixedSectionText css="mixedtext" title={'browse'} subtitle={'all products'}/>
                 <div id='browseall' className='flex flex-col lg:flex-row'>
-                    <div id='bycause'><ButtonBrowse title={'by cause'} /></div>
```

Comment:
We should probably break these down into components since it is being reused all over this page

Reply:
done! new request is below